prime-router/metadata/schemas/WATERS/direct-covid-19.schema

```diff
@@ -1,94 +1,96 @@
 ---
```

Comment:
This is a draft schema, not in use yet, meant to mimic the Waters incoming schema.   Its elements were taken from the Waters programmers guide, as well as looking at sample data.